doc/api/crypto.md

```diff
@@ -2247,3 +2258,5 @@ the `crypto`, `tls`, and `https` modules and are generally specific to OpenSSL.
 [initialization vector]: https://en.wikipedia.org/wiki/Initialization_vector
 [stream-writable-write]: stream.html#stream_writable_write_chunk_encoding_callback
 [stream]: stream.html
+[OpenSSL options]: #openssl-options
+[`stream.transform` options]: stream.html#stream_new_stream_transform_options
```

Comment:
Can you sort the references ASCIIbetically? See the references that are already there.